target/linux/ramips/image/mt7620.mk

```diff
@@ -350,6 +350,14 @@ define Device/rt-n14u
 endef
 TARGET_DEVICES += rt-n14u
 
+define Device/rt-ac51u
+  DTS := RT-AC51U
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
+  DEVICE_TITLE := Asus RT-AC51U
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-ehci kmod-usb-ledtrig-usbport
```

Comment:
You can drop the ```kmod-usb-ledtrig-usbport``` here. You don't have (and don't setup) a LED which can be used for USB status indication (albeit you want to use the activity for usb).